projects/plugins/protect/src/js/components/threats-list/index.jsx

```diff
@@ -132,21 +137,23 @@ const ThreatsList = () => {
 													fixableList.length
 												) }
 											</Button>
-											<OnboardingPopover
-												id="paid-fix-all-threats"
-												position={ isSm ? 'bottom right' : 'middle left' }
-												anchor={ showAutoFixersPopoverAnchor }
-											/>
+											{ ! scanning && (
+												<OnboardingPopover
+													id="paid-fix-all-threats"
+													position={ isSm ? 'bottom right' : 'middle left' }
+													anchor={ showAutoFixersPopoverAnchor }
+												/>
+											) }
+											<ScanButton ref={ setDailyAndManualScansPopoverAnchor } />
+											{ ! scanning && (
+												<OnboardingPopover
+													id="paid-daily-and-manual-scans"
+													position={ isSm ? 'bottom left' : 'middle left' }
```

Comment:
```suggestion
													position={ isSm ? 'bottom left' : 'middle right' }
```
<img width="318" alt="Screenshot 2024-10-16 at 6 10 45 PM" src="https://github.com/user-attachments/assets/eb70c62a-82a8-4a3c-9d13-188669613d67">

Reply:
This might be a pre-existing issue - on narrower screens the popover goes out of the screen. I'm hoping the above code suggestion will move it to the right side of the reported date?

Reply:
Good catch - the line you mention changing though relates to the `dailyAndManualScansPopoverAnchor` but based on the image I think you are referring to the free `dailyScansPopoverAnchor` in the `ScanAdminSectionHero`. Due to the length of the subheading content there it ended up jumping to the far right of that block of text provided the `middle right` position - I will see what I can do about forcing to display immediately afterwards on the right.